package.json

```diff
@@ -51,5 +51,15 @@
     "style-loader": "^0.13.0",
     "webpack": "2.2.0",
     "webpack-dev-server": "2.2.0"
+  },
+  "browserslist": {
```

Comment:
can you create instead a `.browserslistrc` file in the root instead of updating `package.json`?